_posts/newsletters/2019-10-24-newsletter.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: October 2019 US-RSE Newsletter
+subtitle: US-RSE Newsletter
+category: newsletter
```

Comment:
We should stay away from tags with spaces until we can be sure to url_encode all the places they might be linked. All lowercase with hyphens would work for now.

Reply:
Easy fix.